clients/mobile/src/LocationService/LocationService.ts

```diff
@@ -28,7 +51,7 @@ export class LocationService {
       const hasFirstResult = Object.prototype.hasOwnProperty.call(response, '0')
       if (hasFirstResult) {
         const { lat, lon } = response['0']
-        return { lat, lon }
+        return { latitude: +lat, longitude: +lon }
```

Comment:
is the unary operator safe? or should we NUMBER it?

Reply:
Yes vai, it is safe for converting strings to numbers and it is faster than `Number()` also but there is issue like if somehow we get "ABC" and want to convert to a number  then it will give `NaN` so that's the downside of this and I think we should create a custom function for converitng string to number.

Reply:
Create a function for converting string to number

```
export function stringToNumber(str: string): number {
  const num = Number(str)
  if (isNaN(num)) {
    throw new Error('Invalid number string')
  }
  return num
}

```